src/Pages/SignIn/Google.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom';
+
+const withRouter = (Component) => {
+  const Wrapper = (props) => {
+    const navigate = useNavigate();
+    
+    return (
+      <Component
+        navigate={navigate}
+        {...props}
+        />
+    );
+  };
+  
+  return Wrapper;
+};
+
+class Google extends React.Component {
+
+  constructor(props) {
+    super(props)
+    window.customCallback = this.customCallback.bind(this)
+    this.customCallback=this.customCallback.bind(this);
+  }
+  
+  customCallback(response) {
+    localStorage.setItem("token", response.credential)
```

Comment:
We need to update token name to myKitchenAppToken.

Reply:
Ok I'll change that to be 
   localStorage.setItem("myKitchenAppTolen", response.credential)